stl/inc/experimental/random

```diff
@@ -0,0 +1,50 @@
+// random experimental header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _EXPERIMENTAL_RANDOM_
+#define _EXPERIMENTAL_RANDOM_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+
+#include <random>
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+namespace experimental {
+    inline namespace fundamentals_v3 {
+
+        inline default_random_engine& _Engine() {
+            thread_local default_random_engine _Eng{random_device{}()};
+            return _Eng;
+        }
+
+        template <class _IntType>
+        _IntType randint(const _IntType _Min, const _IntType _Max) {
+            _RNG_REQUIRE_INTTYPE(randint(), _IntType);
+            _STL_ASSERT(_Min <= _Max, "invalid min and max arguments for randint()");
+            return uniform_int_distribution<_IntType>{_Min, _Max}(_Engine());
+        }
+
+        inline void reseed(const default_random_engine::result_type _Value = random_device{}()) {

```

Comment:
I believe this should be split into two overloads, and `reseed()` with zero arguments should initialize the whole state.

Reply:
Frag, sorry about that. I asked for those two to be merged but did not oversee the required divergence due to the unpredictable requirement.